tests/api/test_api_db.py

```diff
@@ -0,0 +1,28 @@
+""" Tests for datamegh.api.db. """
+
+import pytest
+
+from mock import patch
+
+from datamegh.api import db
+from datamegh.util.db import PG
+
+
+@patch("datamegh.api.db.pyodbc")
+def test_connect(m_pyodbc):
+    """
+    Test it invokes pyodbc connect
+    generating a connection string.
+    """
+
+    connection = db.connect(
```

Comment:
No need to declare connection variable. It is not being used

Reply:
Good catch.